### PR TITLE
[rime] add paging callback

### PR DESCRIPTION
**Pull request by osfans**

### CODE REVIEW

File: src/fcitx-rime.c [hunk 1]
```diff
@@ -364,6 +365,15 @@ INPUT_RETURN_VALUE FcitxRimeGetCandWord(void* arg, FcitxCandidateWord* candWord)
     return retVal;
 }
 
+boolean FcitxRimePaging(void* arg, boolean prev) {
+    FcitxRime *rime = (FcitxRime *)arg;
+    boolean result = RimeProcessKey(rime->session_id, prev ? FcitxKey_Page_Up : FcitxKey_Page_Down, 0);
```
Comment (wengxt) [inline src/fcitx-rime.c]: I doubt whether this will always work. Rime allows you to change the key binding.

Comment (osfans) [inline src/fcitx-rime.c]: According to https://github.com/rime/librime/blob/develop/src/gear/selector.cc#L32, XK_Prior/XK_Page_Up always do page up and  XK_Next/XK_Page_Down always do page down.
But maybe key binding works before selector?

@lotem Any suggestions to do page up/down? Add new API or just use RimeProcessKey?

Comment (wengxt) [inline src/fcitx-rime.c]: Seems ibus-rime uses RimeProcessKey, I assume that's somewhat acceptable solution.

**Comment by osfans** [comment]
According to default key_binder, it's acceptable to do paging by sending Page_Up / Page_Down.
```yaml
key_binder:
  bindings:
    # Emacs style
    - { when: composing, accept: Alt+v, send: Page_Up }
    - { when: composing, accept: Control+v, send: Page_Down }
    # paging keys
    - { when: composing, accept: ISO_Left_Tab, send: Page_Up }
    - { when: composing, accept: Shift+Tab, send: Page_Up }
    - { when: composing, accept: Tab, send: Page_Down }
    - { when: paging, accept: minus, send: Page_Up }
    - { when: has_menu, accept: equal, send: Page_Down }
    - { when: paging, accept: comma, send: Page_Up }
    - { when: has_menu, accept: period, send: Page_Down }
```

**Pull request merged**